docs/notebooks/tutorials/basic_optimization.ipynb

```diff
@@ -1061,7 +1061,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "desc-env",
```

Comment:
did you forgot to switch back to the python kernel

Reply:
Are we doing that? This happened automatically, I haven't noticed.

Reply:
yea you should, I think, technically this change is fine but if any local user has renamed the kernel differently than `desc-env`, they would not be able to run locally, or if your kernel was not `desc-env`, the notebook would not run on CI

Reply:
Is there an easy way to do this? I thought we were waiting for #1215. When I change the kernel to my default Python, it shows `"display_name": "Python 3"` not `"display_name": "Python 3 (ipykernel)"`. There are a lot of notebooks with different kernel names, I can also change them but it is really cumbersome for future since this change happens automatically if you just click on the notebook.

Reply:
I believe the problem with CI was only happening after cluster use (for some reason). If you look at free boundary tutorial etc, they also have `desc-env` as kernel.

Reply:
I use conda environments that are not named `desc-env`

Reply:
See comment below for `coil_stage_two_optimization.ipynb`. It previously had `desc-env-latest` but working fine.